### ISSUE TITLE
[400 AUTH_BYTES_INVALID] Telegram File (Some) Fail

### ISSUE THREAD

**Issue by anonymies-bozzku** [viperadnan-git/google-drive-telegram-bot#100]
there's problem when forwarding files to the bot.. some of them able to download n upload, but some can't
if I download that file and direct send.. there's no problem
it seems like something need to do with the pyrogram.. I guess

![Screenshot (31)](https://user-images.githubusercontent.com/87874591/158225371-b66878a3-aa0d-45e8-9d27-83dfe95ee083.png)

**Comment by shadowninja024** [comment]
Yes, im using his code of older version in one of my bot thats working perfectly.
The bots on bew code have this problem.
Hope some one pull up this issue and solve

**Comment by thethor12** [comment]
Which old code
